src/inspector_socket_server.cc

```diff
@@ -376,11 +385,25 @@ void InspectorSocketServer::SendListResponse(InspectorSocket* socket) {
 
 bool InspectorSocketServer::Start() {
   CHECK_EQ(state_, ServerState::kNew);
-  sockaddr_in addr;
+  struct addrinfo hints;
+  memset(&hints, 0, sizeof(struct addrinfo));
```

Comment:
`sizeof(hints)` or just `struct addrinfo hints()`.

I will admit I don't quite understand why you pass the port number if you aren't interested in service name lookups (AI_NUMERICSERV.)